woocommerce/payment-gateway/payment-tokens/class-sv-wc-payment-gateway-payment-tokens-handler.php

```diff
@@ -584,6 +584,35 @@ public function update_tokens( $user_id, $tokens, $environment_id = null ) {
 	}
 
 
+	/**
+	 * Updates a single legacy token in user meta.
+	 *
+	 * @internal
```

Comment:
`@internal` tag not needed if the method is accessed from elsewhere outside the class - let's keep this for hook callbacks